src/main/java/banking/BankingSystem.java

```diff
@@ -0,0 +1,185 @@
+package banking;
+
+import banking.dao.AccountDAO;
+import banking.model.Account;
+import banking.model.Card;
+import java.util.Scanner;
+import static java.lang.System.exit;
+
+public class BankingSystem {
+
+    private int id;
+    private int amountSend;
+    private int choose;
+    private boolean isLogin;
+    private CardValidator cardValidator;
+    private AccountDAO accountDAO;
+    private Scanner scanner;
+    private String card;
```

Comment:
Banking System shouldn't keep the Account and Card state. 
e.g. 
```
private int id;
private String card;
```
Can you refactor that?

Reply:
fixed

Reply:
Why choose variable is described as an instance variable?

Reply:
fixed